src/EnergyPlus/SimulationManager.cc

```diff
@@ -234,6 +317,14 @@ namespace SimulationManager {
         state.dataGlobal->DoingSizing = true;
         SizingManager::ManageSizing(state);
 
+        std::thread first;
+        if (!state.dataHeatBal->Zone.empty()) {
+            if (state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, "EnhancedInteractivity") > 0) {
+                state.dataGlobal->printConsoleOutput = false;
+                first = std::thread(enhancement, &state);
+            }
+        }
```

Comment:
New capability does run on a background thread while the simulation is running, of course.